app/models/concerns/justifiable.rb

```diff
@@ -5,16 +5,24 @@ module Justifiable
     attr_accessor :ticket_number
     attr_reader :ticket_url
 
+    before_validation :strip_octothorpe
     validates :ticket_number,
               presence: true,
-              numericality: { only_integer: true },
+              numericality: { only_integer: true,
+                              message: "may begin with an # and otherwise contain only numbers." },
```

Comment:
It'd be nice to [i18n](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards#user-content-Validation_errors) this error message by making it use a custom error message i18n key:

```suggestion
                              message: :numeric_with_optional_hash },
```

And then defining the translation for that key in `config/locales/models/en.yml` under `activerecord`:
```
  errors:
    messages:
      numeric_with_optional_hash: 'may begin with an # and otherwise contain only numbers.'
```